apps/screloc/main.cpp

```diff
@@ -270,35 +273,56 @@ class Reloc : public Client::Application {
 						}
 					}
 
+					bool relocated = false;
 					try {
 						org = process(org.get());
+						relocated = true;
 					}
 					catch ( std::exception &e ) {
 						SEISCOMP_ERROR("  + processing failed - %s", e.what());
+						relocated = false;
```

Comment:
No necessary as `false` is default.

Reply:
I know, but I like pedantic/obvious code.  The compiler will remove it anyway.